src/lcals/FIRST_MIN-Cuda.cpp

```diff
@@ -80,24 +80,38 @@ void FIRST_MIN::runCudaVariantImpl(VariantID vid)
 
        FIRST_MIN_MINLOC_INIT;
 
-       MyMinLoc* dminloc;
-       cudaErrchk( cudaMalloc( (void**)&dminloc, sizeof(MyMinLoc) ) );
+       const size_t grid_size = RAJA_DIVIDE_CEILING_INT(iend, block_size);
+       MyMinLoc** dminloc;
```

Comment:
Just seemed easier to do with pointers since we're passing them to kernels. But, yeah we could do it as fixed length arrays. We know the dims of the dminloc array.

Reply:
@MrBurmark should be cleaned up now